### DISCUSSION TITLE
Architecture and features

### DISCUSSION THREAD

**Discussion by ghost** [mir-group/nequip#166] [General]
Hello,
What would be happened if you skip training radial network?
Why number of features for toluene is 32? And what are they?
Would you please elaborate more on embedding step? And did you use adjacency matrix?
Could the interatomic potential be used for NVE and NPT simulations?

Thank you in advance!

**Comment by simonbatzner** [comment]
Hi @Zoread, while we're happy to answer all of these questions, we'd ask you to read the relevant literature first (we cite the relevant works in our paper), the answers to your questions can be found there. If after that you still have questions, please don't hesitate to reach out. 

Best, 
Simon

**Reply by simonbatzner** [reply]
Closing this for now.
